sympy/simplify/simplify.py

```diff
@@ -31,6 +31,7 @@
 from sympy.simplify.sqrtdenest import sqrtdenest
 from sympy.simplify.trigsimp import trigsimp, exptrigsimp
 from sympy.utilities.iterables import has_variety, sift
+from sympy.functions.elementary.complexes import Abs
```

Comment:
This should combine with line 18 where there is already an import from this module

Reply:
ok, sure